examples/vertx-opentracing/pom.xml

```diff
@@ -62,4 +62,16 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${version.maven-deploy-plugin}</version>
```

Comment:
A small technicality: unless you have a good reason to override the version of the plugin managed in the parent https://github.com/hawkular/hawkular-parent-pom/blob/master/pom.xml#L530 you do not need to have the `<version>` here explictily. If your IDE cannot resolve the plugin without version, rather add the explicit `<groupId>org.apache.maven.plugins</groupId>`